src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-OpportunitySideCard/Components/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import * as S from '../styles';
+import { Opportunity, Cardtypes } from '../types';
+import { Paragraph } from '../../../ignitus-Atoms/typography';
+import ShareIcon from '../../../ignitus-Assets/ignitus-Images/img-Png/share.png';
+
+export const InfoCard = ({ location, commitment }: Cardtypes) => (
+  <S.Container>
+    <Card title="Location" description={location} />
+    <Card title="Commitment" description={commitment} />
+    <S.ButtonsWrapper>
+      <S.Button size="large" category="primary">
+        {' '}
+        Apply{' '}
```

Comment:
Can you please add the dynamic feature as well. On clicking the apply button, a message will be displayed saying `Applied successfully` and the button will change to a button with a checkmark icon and text as Applied. YOu can see the screens in Figma for more reference.

Reply:
Okay!

Reply:
I checked the PR, I guess you missed this.